src/resources/style.css

```diff
@@ -9,13 +9,13 @@
     line-height: 1.1;
     display: flex;
     flex: auto;
-    /*justify-content: space-between;*/
```

Comment:
Do you know why this was initially commented out? Might be a good idea to remove altogether instead of uncommenting to prevent unintended side-effects in the UI.

Reply:
No, but without it the styling is weird. (all mushed agains one side of the container)
This extension is old enough that flex was still in its infancy when it was written :P 

I'll leave a note for tests to check the styling.